packages/apollo-language-server/src/utilities/graphql.ts

```diff
@@ -136,8 +136,15 @@ export function withTypenameFieldAddedWhereNeeded(ast: ASTNode) {
       }
     },
     leave(node: ASTNode) {
-      if (!(node.kind === "Field" || node.kind === "FragmentDefinition"))
+      if (
+        !(
+          node.kind === "Field" ||
+          node.kind === "FragmentDefinition" ||
+          node.kind === "InlineFragment"
```

Comment:
I know you're just perpetuating the pattern that was already here, but thoughts on replacing these string literals (e.g. `"Field"`, `"FragmentDefinition"`, etc.) with the `graphql-js` "`Kind`" constants, for example, [`Kind.FRAGMENT_DEFINITION`](https://github.com/graphql/graphql-js/blob/8c96dc8276f2de27b8af9ffbd71a4597d483523f/src/language/kinds.js#L28).

Reply:
Definitely. Good call!

Reply:
Resolved in 6e61408